.travis.yml

```diff
@@ -33,12 +33,15 @@ install:
   # Install the package and dependencies.
   - pip install -e .
 
-# command to run tests, e.g. python setup.py test
 script:
+  # Run our test suite and report coverage.
   - coverage erase
   - coverage run --source . setup.py test
   - coverage report -m
 
+  # Do some basic linting of the code.
+  - flake8 .
```

Comment:
This runs on all of our code including the library, the tests, and our `setup.py` file.